node/main.cpp

```diff
@@ -120,9 +120,32 @@ int main(int argc, const char **argv, const char **env) {
 
   soralog::util::setThreadName("kagome");
 
-  auto logging_system = std::make_shared<soralog::LoggingSystem>(
-      std::make_shared<kagome::log::Configurator>(
-          std::make_shared<libp2p::log::Configurator>()));
+  // Logging system
+  auto logging_system = [&] {
+    auto custom_log_config_path =
+        kagome::log::Configurator::getLogConfigFile(argc - 1, argv + 1);
+    if (custom_log_config_path.has_value()) {
+      if (not std::filesystem::is_regular_file(
+              custom_log_config_path.value())) {
+        std::cerr << "Provided wrong path to config file of logging\n";
```

Comment:
```suggestion
        std::cerr << "Provided wrong path to logger config file\n";
```
wording + why not to flush stream via std::endl?

Reply:
`std::endl` calls `std::cerr.flush()`. clang-tidy warns to it.
We doing call of `flush()` once at exit.